src/Compilers/CSharp/Portable/Lowering/MethodToClassRewriter.cs

```diff
@@ -251,31 +251,6 @@ private bool TryGetHoistedField(Symbol variable, [NotNullWhen(true)] out FieldSy
             return false;
         }
 
-        public override BoundNode VisitAwaitableInfo(BoundAwaitableInfo node)
```

Comment:
This API was pulled up to the base type in https://github.com/dotnet/roslyn/pull/78517.